db/migrations/0003_player_created_at_alter_player_guild.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-12-28 10:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_player_guild_alter_player_race_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='player',
+            name='created_at',
+            field=models.DateTimeField(auto_now=True),
```

Comment:
The `created_at` field is set with `auto_now=True`, which updates the field to the current timestamp every time the Player object is saved. If the intention is to only set this field when the object is created, consider using `auto_now_add=True` instead.